docs/tutorials/typescript/work-queue-slack-app/index.mdx

```diff
@@ -0,0 +1,991 @@
+
+### Functional requirements
+
+**What are the functional requirements of the application?**
+
+1. Using a Slash command, a user can submit a work request.
+2. Using a Slash command, a user can see all the work requests for the channel.
+3. When viewing all the work requests for the channel, a user can click a button to claim a work request.
+4. When viewing all the work requests for the channel, a user can click a button to mark a work request as complete.
+5. Using a Slash command, a user can delete the Work Queue for the channel.
+
+![App functionality demo](./images/work-queue-demo.gif)
+
+## Create a new Slack App configuration
+
+Go to https://api.slack.com/apps?new_app=1 and create an app "From scratch." Name the app whatever you want, and select a workspace that you intend to install it to.
+
+Once you've done this, click "Create App."
+
+### Enable Socket Mode
+
+Under Settings in the navigation bar, select Socket Mode and toggle ON socket mode for this application.
+
+![Enable Socket Mode](./images/enable-sockets.png)
+
+When you enable Socket Mode, Slack gives you an application token that you will need when you start the app.
+This token will typically start with `xapp`.
+It is located under App-Level Tokens in Basic Information.
+
+### Create a Slash Command
+
+Select "Slash Commands" in the sidebar navigation, and click "Create New Command".
+Add a new command for `/workqueue`.
+Since you have Socket Mode enabled, you won't need to add a request URL.
+
+![Create a Slash Command](./images/enable-slash-command.png)
+
+### Subscribe to Events
+
+Under Features in the navigation bar, select Event Subscriptions and ensure Enable Events is turned ON.
+The navigate to the section under "Subscribe to bot events" and add subscriptions for:
+
+- `message.groups`
+- `message.im`
+- `message.channels`
+
+![Subscribe to bot events](./images/subscribe-to-bot-events.png)
+
+Once you've done this, click "Save Changes"
+
+### Set OAuth & Permissions
+
+Under Features in the navigation bar, go to OAuth & Permissions.
+
+You'll need to enable the following Bot Token Scopes:
+
+- `channels:history`
+- `channels:read`
+- `chat:write`
+- `groups:history`
+- `groups:read`
+- `im:history`
+- `mpim:history`
+
+### Install App
+
+After you've done this, install the app in Slack by going to Install App under Settings in the navigation bar.
+
+### Get credentials
+
+You will need **Signing Secret** and **App Level Token** from the Basic Information page.
+And you will need the **Bot User OAuth Token** from the Install App page, which is also available on the OAuth & Permissions page.
+
+## Create TypeScript projects for the Slack App
+
+You need two TypeScript projects for this Slack application.
+You may have already created the first one for the Temporal Application in the [Prerequisite steps](/getting_started/typescript/dev_environment/).
```

Comment:
```suggestion
The first of these can just be named `temporal-application`. If you recently completed the [Set up a local development environment for Temporal and TypeScript](/getting_started/typescript/dev_environment/) tutorial, you can reuse that project structure. The other project should be called `bot`. You can create a new Temporal TypeScript project using `npx`:

npx @temporalio/create@latest ./bot
```

Reply:
I don't know of a binary for creating a Slack app to use with `npx` - 
You need to use `npm init` and `tsc init`...

Reply:
But I do like the rephrase